google/ggrc-core
# GGRC-4019 Cannot unmap Assessment from Issue due to incorrect message is displayed if click Unmap from Issue

# Issue description

Cannot unmap Assessment from Issue due to incorrect message is displayed if click Unmap from Issue.

# Steps to test the changes

Steps to reproduce:
1. Go to Audit page with a control snapshot
2. Generate Assessment A based on a control snapshot
3. Navigate to Assessment Info panel > Related Issues tab and Raise an Issue for the Assessment A
4. Go to Issue Info page > Assessment tab
5. Refresh the page
6. Navigate to Assessment Info pane and click Unmap from Issue in 3 bb's menu
Actual Result: "Unmapping cannot be performed. Please unmap Issue (new raised Issue_12/7-1) from Assessment version (new Assessment 12/6-8), then mapping with original object will be automatically reverted." message is displayed while clicking Unmap from Issue. Assessment is not unmapped from Assessment.
Expected Result: While clicking Unmap from Issue Info popup is displayed to inform a user that:
- audit will be unmapped;
- snapshot will be unmapped.

# Solution description

If we try to load issue from general info page then relationships is loaded. But, if we try just to load assessment tab then needed relationship isn't loaded.
**Solution:**
We should load the relationship for `target` object and `issue`.

# Sanity checklist




# PR Review checklist
